cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,78 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
+const { faker } = require('@faker-js/faker');
+
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import homePageObject from '../support/pages/home.pageObject';
+import ProfilePageObject from '../support/pages/profile.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
+
+const settingsPage = new SettingsPageObject();
+const homePage = new homePageObject();
+const profilePage = new ProfilePageObject();
+const signInPage = new SignInPageObject();
 
 describe('Settings page', () => {
-  before(() => {
+  let user;
 
+  before(() => {
+    cy.task('db:clear');
   });
 
   beforeEach(() => {
-
+    cy.task('generateUser').then((generateUser) => {
```

Comment:
It is better to use async/await to handle promises. Consider rewriting the code with async/await syntax for better readability and error handling.

Reply:
![image](https://github.com/mate-academy/qa_cypress_e2e_settings_pom_local/assets/84145664/89c10655-f0ad-4df9-b579-f2a418e9307d)